src/muya/lib/contentState/updateCtrl.js

```diff
@@ -408,25 +408,40 @@ const updateCtrl = ContentState => {
         const preInputChar = text.charAt(+offset - 2)
         const postInputChar = text.charAt(+offset)
         /* eslint-disable no-useless-escape */
-        // Not Unicode aware, since things like \p{Alphabetic} or \p{L} are not supported yet
         if (
-          (autoPairQuote && /[']{1}/.test(inputChar) && !(/[a-zA-Z\d]{1}/.test(preInputChar))) ||
-          (autoPairQuote && /["]{1}/.test(inputChar)) ||
-          (autoPairBracket && /[\{\[\(]{1}/.test(inputChar)) ||
-          (autoPairMarkdownSyntax && /[*_]{1}/.test(inputChar))
+          (event.inputType.indexOf('delete') === -1) &&
+          (inputChar === postInputChar) &&
+          (
+            (autoPairQuote && /[']{1}/.test(inputChar)) ||
+            (autoPairQuote && /["]{1}/.test(inputChar)) ||
+            (autoPairBracket && /[\}\]\)]{1}/.test(inputChar)) ||
+            (autoPairMarkdownSyntax && /[*_]{1}/.test(inputChar))
+          )
         ) {
-          text = BRACKET_HASH[event.data]
-            ? text.substring(0, offset) + BRACKET_HASH[inputChar] + text.substring(offset)
-            : text
-        }
-        /* eslint-enable no-useless-escape */
-        if (/\s/.test(event.data) && preInputChar === '*' && postInputChar === '*') {
           text = text.substring(0, offset) + text.substring(offset + 1)
+          this.cursor = lastCursor = { start, end }
```

Comment:
Why need add this line code?

Reply:
This is because last implement used `particalRender` function, now yes It not needed.